x-pack/plugins/security_solution/common/experimental_features.ts

```diff
@@ -92,6 +92,10 @@ export const allowedExperimentalValues = Object.freeze({
    * Enables expandable flyout in create rule page, alert preview
    */
   expandableFlyoutInCreateRuleEnabled: true,
+  /**
+   * Enables expandable flyout in timeline
+   */
+  expandableTimelineFlyoutEnabled: true,
```

Comment:
Is there a way to disable the flag outside of this file? To me it's a little risky to default true, since once merged it will take affect in serverless

Reply:
good point, I can switch it back to `false` by default!